CheckIn/Manager/ReadMe.md

```diff
@@ -0,0 +1,122 @@
+
+
+# Advanced Check-In Monitor
+_Tested/Supported in Rock Version:  8.0-12.0_    
+_Released:  12/31/2018_   
+_Updated:  3/16/2020_   
+
+## Summary
+
+This block is an advanced version of the core Check-in Manager block. It adds the following functions:
+
+- Ability to view and reprint labels
+- Ability to move a person who is checked in to a new location or group
+- Ability to check-out a person
+- Ability to move all people in a location to another location or group
+
+
+
+Quick Links:
+
+- [What's New](#whats-new)
+- [Using the Block](#using-the-block)
+- [Block Properties](#block-properties)
+
+
+## What's New
+
+The following new goodness will be added to your Rock install with this plugin:
+
+- **New Block**: *Check-in Manager Locations*
+
+
+
+## Using the Block
+
+**You will need to swap out the core Locations block on the Check-in Manager page for the KFS Locations block.**
+
+On the check-in manager page, choose the check-in area, group and location that you would like to manage. When you get to the list of people who are checked in, you will notice a few differences. 
+
+
+
+![Images/PersonOptions](https://user-images.githubusercontent.com/81330042/115042860-7a103700-9e99-11eb-8105-48df431b7246.png)
+
+
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;1</span> **Move All** Allows you to move everyone checked into this location to a new location and/or group. Imagine you have a spill on the carpet in one of your rooms. You want to move all the kids from one room to another so you can clean it up, the move all button gives you an easy way to do that. Your leaders will still have an accurate count and list of who should be in which room. 
+>
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;2</span>**View Labels** Allows you to view all the labels that printed when someone checked it. It also gives you the ability to reprint their label with the same security code. This is very helpful when a child rips or spills on their label. You can print another label with the same security code so it will still match the parent's label at pickup. Printing relies on the device having a default printer. 
+>
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;3</span>**Move** Allows you to move just one person to a new location and/or group. 
+>
+> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;4</span>**Checkout** Allows you to checkout the person. 
```

Comment:
```suggestion
> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;1&nbsp;&nbsp;</span>**Move All** Allows you to move everyone checked into this location to a new location and/or group. Imagine you have a spill on the carpet in one of your rooms. You want to move all the kids from one room to another so you can clean it up, the move all button gives you an easy way to do that. Your leaders will still have an accurate count and list of who should be in which room. 
>
> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;2&nbsp;&nbsp;</span>**View Labels** Allows you to view all the labels that printed when someone checked it. It also gives you the ability to reprint their label with the same security code. This is very helpful when a child rips or spills on their label. You can print another label with the same security code so it will still match the parent's label at pickup. Printing relies on the device having a default printer. 
>
> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;3&nbsp;&nbsp;</span>**Move** Allows you to move just one person to a new location and/or group. 
>
> <span style="padding-left: 30px; margin-right: 10px; width: .8em;background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">&nbsp;&nbsp;4&nbsp;&nbsp;</span>**Checkout** Allows you to checkout the person. 
```